x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```
@@ -63,6 +63,117 @@ setup:
   - match: { "test-index.mappings.properties.sparse_field.model_settings.task_type": sparse_embedding }
   - length: { "test-index.mappings.properties.sparse_field": 3 }
 
+---
+"Field caps set include_empty_field to false with sparse embedding":
+
+  - requires:
+      cluster_features: "gte_v8.16.0"
+      reason: field_caps support for semantic_text added in 8.16.0
+
+  - do:
+      field_caps:
+        include_empty_fields: false
+        index: test-index
+        fields: "*"
+
+  - match: { indices: [ "test-index" ] }
+  - is_false: fields.sparse_field
+  - is_false: fields.dense_field
```

`not_exists` is probably a better assertion here:

```suggestion
  - not_exists: fields.sparse_field
  - not_exists: fields.dense_field
```